libraries/standard/json/cbmc/proofs/skipString/Makefile

```diff
@@ -0,0 +1,13 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+HARNESS_FILE=skipString_harness
+
+CBMC_MAX_BUFSIZE=14
```

Comment:
why?

Reply:
For some of the proofs, I had to titrate a value that was large enough for complete coverage without tripping unwinding assertions and small enough to not exhaust memory.  I then tried to use the value in several proofs, but some had to be reduced further due to memory.

Reply:
Add a comment explaining this in the file.

Reply:
added